topics/xampp/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: xampp-server, xampp-control-panel
+created_by: Apache Friends
+display_name: XAMPP
+logo: xampp.png
+released: 2002
+short_description: XAMPP is the combination of the Apache web server, the MySQL database and scripting langs.
+topic: xampp
+url: https://www.apachefriends.org/de/index.html
+wikipedia_url: https://de.wikipedia.org/wiki/XAMPP
+---
+XAMPP is a program package of free software - originally mainly in the environment of the LAMP system. XAMPP enables the installation and configuration of the web server Apache with the database MariaDB (replaced MySQL in October 2015) or SQLite and the scripting languages ​​Perl and PHP (with PEAR). The X ("cross-platform") stands for the different operating systems on which it can be used. XAMPP also includes other useful tools such as the FTP server ProFTPD or FileZilla Server, the mail server Mercury, phpMyAdmin, Webalizer and OpenSSL. From version 1.7.4 onwards, the Windows variant also includes the web server and web container Apache Tomcat 7, which enables the execution of JavaServer Pages and Java Servlets.
```

Comment:
Tiny editorial suggestion for some punctuation while I'm making suggestions anyway 😄 

```suggestion
XAMPP is a program package of free software, originally mainly in the environment of the LAMP system. XAMPP enables the installation and configuration of the web server Apache with the database MariaDB (replaced MySQL in October 2015) or SQLite and the scripting languages ​​Perl and PHP (with PEAR). The X ("cross-platform") stands for the different operating systems on which it can be used. XAMPP also includes other useful tools such as the FTP server ProFTPD or FileZilla Server, the mail server Mercury, phpMyAdmin, Webalizer and OpenSSL. From version 1.7.4 onwards, the Windows variant also includes the web server and web container Apache Tomcat 7, which enables the execution of JavaServer Pages and Java Servlets.
```